app/components/maps/SidebarReplays.tsx

```diff
@@ -62,6 +67,44 @@ const SidebarReplays = ({
         return uniques.sort().map((val) => ({ text: val, value: val }));
     };
 
+    const deleteReplayFile = async (replay: ExtendedFileResponse) => {
+        try {
+            await deleteReplay(replay);
+            await onRefreshReplays();
+            message.success('Replay deleted!');
+        } catch (e) {
+            message.error('Could not delete replay, please try again.');
```

Comment:
Not sure if "try again" is appropriate here tbh 😄 I assume if it went wrong once, it probably won't work the second time.

Reply:
Yeah, I agree. Will change it.